spec/models/service_spec.rb

```diff
@@ -53,4 +53,52 @@
       expect(service.required_tiers("all")).to eq([2, 3, 1])
     end
   end
+
+  describe "#duplicate!" do
+    it "duplicates the service" do
+      service = create(:service, :all_tiers, lgsl_code: 1)
+
+      interaction1 = create(:interaction, lgil_code: 1, label: "interaction1")
+      service_interaction1 = create(:service_interaction, service: service, interaction: interaction1)
+      create(:link, service_interaction: service_interaction1, url: "https://example.com")
+      create(:link, service_interaction: service_interaction1, url: "https://example.com/test")
+      create(:link, service_interaction: service_interaction1, url: "https://example.com/test/again")
```

Comment:
I don't think you need the URL's on these. You can shorten it to `create_list(:link, 3 service_interaction: service_interaction1)`

Reply:
Thanks - fixed up as per your suggestion